### DISCUSSION TITLE
Welcome to Rails UI discussions

### DISCUSSION THREAD

**Discussion by justalever** [getrailsui/railsui#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other community members. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To start, comment below with an introduction and tell us about what you do with this community.

**Comment by ghost** [comment]
Hello!

**Comment by mostlyerror** [comment]
Hi, congrats on the launch! Love the idea.

**Comment by DogAndCode** [comment]
Congrats for this idea!
